.vscode/settings.json

```diff
@@ -65,5 +65,8 @@
   "stylelint.additionalDocumentSelectors": ["typescript", "typescriptreact"],
   "todohighlight.exclude": ["node_modules/**", ".vscode/**", "externals/**"],
   "tslint.autoFixOnSave": true,
-  "typescript.format.enable": false
+  "typescript.format.enable": false,
+  "[json]": {
+    "editor.formatOnSave": false
```

Comment:
For whatever reason, rules like "require trailing comma" would auto-fix in `.json` files making it impossible to produce correct JSON.